README.md

```diff
@@ -142,6 +142,11 @@ The tests can be run via `cargo`:
 
 `cargo test`
 
+If you run into test failures, try these remedies:
+
+- unset CT_API_KEY
+- `cargo build`
+
```

Comment:
Is this an old comment? I think that I have fixed these instances. If not, we probably want to unset CLOUDTRUTH_API_KEY and CLOUDTRUTH_SERVER_URL.